src/main/java/utils/Calculator.java

```diff
@@ -0,0 +1,48 @@
+package utils;
+
+import java.util.List;
+
+public class Calculator {
+    private static final String PLUS_SYMBOL = "+";
+    private static final String MINUS_SYMBOL = "-";
+    private static final String MULTIPLY_SYMBOL = "*";
+    private static final String DIVIDE_SYMBOL = "/";
+
+    public static int evaluateExpression(List<String> expressions) {
+        int result = InputView.toInt(expressions.get(0));
+
+        for (int i = 1; i < expressions.size(); i += 2) {
+            String operator = expressions.get(i);
+            int nextNumber = InputView.toInt(expressions.get(i + 1));
+
+            result = calculate(result, nextNumber, operator);
+        }
+
+        return result;
+    }
+
+    public static int calculate(int number1, int number2, String operator) {
+        if (operator.equals(PLUS_SYMBOL)) return sum(number1, number2);
+        if (operator.equals(MINUS_SYMBOL)) return subtract(number1, number2);
+        if (operator.equals(MULTIPLY_SYMBOL)) return multiply(number1, number2);
+        if (operator.equals(DIVIDE_SYMBOL)) return divide(number1, number2);
```

Comment:
```suggestion
        switch (operator) {
            case PLUS_SYMBOL:
                return sum(number1, number2);
            case MINUS_SYMBOL:
                return subtract(number1, number2);
            case MULTIPLY_SYMBOL:
                return multiply(number1, number2);
            case DIVIDE_SYMBOL:
                return divide(number1, number2);
        }
```
- 이런 경우 switch 문을 활용하는건 어떨까요